ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
-sunbird_custodian_tenant_name="{{sunbird_custodian_tenant_name}}"
-sunbird_custodian_tenant_description="{{sunbird_custodian_tenant_description}}"
-sunbird_custodian_tenant_channel="{{sunbird_custodian_tenant_channel}}"
-sunbird_root_user_firstname="{{sunbird_root_user_firstname}}"
-sunbird_root_user_lastname="{{sunbird_root_user_lastname}}"
-sunbird_root_user_username="{{sunbird_root_user_username}}"
-sunbird_root_user_password="{{sunbird_root_user_password}}"
-sunbird_root_user_email="{{sunbird_root_user_email}}"
-sunbird_root_user_phone="{{sunbird_root_user_phone}}"
+sunbird_custodian_tenant_name="{{sunbird_init_custodian_tenant_name}}"
+sunbird_custodian_tenant_description="{{sunbird_init_custodian_tenant_description}}"
+sunbird_custodian_tenant_channel="{{sunbird_init_custodian_tenant_channel}}"
+sunbird_root_user_firstname="{{sunbird_init_admin_user_firstname}}"
```

Comment:
this is not the root user, please rename to `sunbird_custodian_admin_*` here and below